<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="navi_mirror_start">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1080</width>
      <height>1920</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1080</width>
      <height>1920</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1080</width>
      <height>1920</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="background.qrc">:/background/background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="map">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>610</width>
      <height>1300</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="qr">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>370</width>
      <height>370</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="description_gotohome">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1600</y>
      <width>1080</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="description.qrc">:/description/description_gotohome.png</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="background.qrc"/>
  <include location="button.qrc"/>
  <include location="description.qrc"/>
  <include location="mic.qrc"/>
 </resources>
 <connections/>
</ui>
